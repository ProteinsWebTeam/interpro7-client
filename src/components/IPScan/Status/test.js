// @flow
import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';

import { IPScanStatus } from '.';

const renderer = new ShallowRenderer();

describe('<IPScanStatus />', () => {
  test('should render', () => {
    renderer.render(
      <IPScanStatus
        jobs={[]}
        search={{}}
        defaultPageSize={20}
        updateJobStatus={() => {}}
      />,
    );
    expect(renderer.getRenderOutput()).toMatchSnapshot();
  });
});
