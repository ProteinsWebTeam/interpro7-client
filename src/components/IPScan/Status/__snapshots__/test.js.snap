// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<IPScanStatus /> should render 1`] = `
<div
  className="row columns"
>
  <h3
    className="light"
  >
    Your InterProScan Search Results
  </h3>
  <Memo(Loadable)
    data={
      Object {
        "description": "The status of all the requested InterProScan searches",
        "name": "Your InterProScan searches",
      }
    }
    processData={[Function]}
  />
  <div
    className="button-bar"
  >
    <div
      className="button-group"
    >
      <GoToNewSearch />
      <Memo(Connect(RefreshButton)) />
    </div>
    <Memo(Connect(ImportResultSearch)) />
  </div>
  <Table
    actualSize={0}
    contentType="result"
    dataTable={Array []}
    query={Object {}}
    rowKey="localID"
    showTableIcon={false}
  >
    <Column
      dataKey="localID"
      renderer={[Function]}
    >
      Results
    </Column>
    <Column
      dataKey="localTitle"
    >
      Title
    </Column>
    <Column
      dataKey="times"
      renderer={[Function]}
    >
      Created
    </Column>
    <Column
      cellClassName="table-center"
      dataKey="status"
      headerClassName="table-center"
      renderer={[Function]}
    >
      Status
    </Column>
    <Column
      cellClassName="table-center font-ml"
      dataKey="localID"
      defaultKey="actions"
      headerClassName="table-center"
      renderer={[Function]}
    >
      Action
    </Column>
  </Table>
</div>
`;
